homeassistant/components/zha/sensor.py

```diff
@@ -507,7 +507,7 @@ def formatter(self, value: int) -> int | float:
 
 @MULTI_MATCH(
     channel_names=CHANNEL_SMARTENERGY_METERING,
-    models={"TS011F"},
+    models={"TS011F", "ZLinky_TIC"},
```

Comment:
Very interesting finding!

Care to take a look at https://github.com/zigpy/zha-device-handlers/pull/1962/files#r1032892143 for firmware v12+?

Reply:
Yes, we must add `ZLinky_TIC_FW_V12` if your PR is merged 🙂

Reply:
Do you think we must create a second model for firmware v12?

If yes, we must add `ZLinky_TIC_FW_V12` if your PR is merged 🙂

Reply:
Ok, I guess I misunderstood how model info works.

I've reverted to using `ZLinky_TIC` as returned by `basic` cluster for `model` attribute.